Please include `py.typed` marker file per PEP 561

https://www.python.org/dev/peps/pep-0561/#packaging-type-information

> Package maintainers who wish to support type checking of their code MUST add a marker file named py.typed to their package supporting typing.

Without this, projects using `gpiod` will fail to run mypy if `ignore_missing_imports = False`

```
gpio.py:7:1: error: Skipping analyzing "gpiod": found module but no type hints or library stubs  [import]
```

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,3 +1,4 @@
 include LICENSE.txt
 include README.md
 include CHANGELOG
+recursive-include py_src py.typed

--- a/setup.cfg
+++ b/setup.cfg
@@ -29,6 +29,3 @@ include_package_data = True
 
 [options.packages.find]
 where = py_src
-
-[options.package_data]
-gpiod = py.typed

